src/main/java/org/zalando/nakadi/service/subscription/state/StreamingState.java

```diff
@@ -237,6 +262,20 @@ private void streamToOutput() {
         }
     }
 
+    private void publishKpi(final String appName, final NakadiKpiPublisher kpiPublisher, final String et,
+                            final long bytes, final long count) {
+        kpiPublisher.publish(
```

Comment:
@v-stepanov would you be fine if I added an "[SLO] [data-streamed]" log here? It would not increase our log amount that much and we could use scalyr to visualize stream data per event type, per subscription, per application and so on.

Reply:
Yes, sure. I was also thinking about something like that. If we calculate this data anyway it makes sense to have it in the logs.